zulipterminal/model.py

```diff
@@ -834,7 +834,7 @@ def fetch_read_receipts(self, message_id: int) -> Optional[List[int]]:
             f"/messages/{message_id}/read_receipts", method="GET"
         )
         if response["result"] == "success":
-            return response["user_ids"]
+            return ", ".join(list(map(self.user_name_from_id, response["user_ids"])))
```

Comment:
I appreciate that this simplifies things from the point of view of the UI code, but note the recent issues we've had with users with identical names - if we pass only the names out, then we lose the ability to determine which users are which.

Somewhere we'll need to identify that the names are the same and perform an operation to distinguish between them (eg. a suffix). That is a UI-related operation in that it will involve (conditional) styling, so isn't model/API related.

Reply:
If this is a standalone bugfix, this could be a separate commit.